src/style.css

```diff
@@ -0,0 +1,57 @@
+/* layout */
+.benefits-section {
+    width: 87vw;
+    height: 655px;
+    background-color: var(--clr-ntrl-min); 
+}
+
+/*  section */
+.benefits-section {
+    position: relative;
+    top: -218px;
+    left: 7.4%;
+}
+
+.benefits-section .benefits-header {
+    color: #000;
+    padding-top: 93px;
+    padding-left: 99px;
+}
+
+.benefits-section .benefits-content {
+    display: flex;
+    width: 88%;
+    justify-content: space-between;
+    margin-top: 120px;   
+    margin-left: 100px;
+}
+
+/* components */
+.benefits-content .card-item {
+    position: relative;
+    width: 26%;
+    height: 280px;
+}
+
+
+.card-item  .card-header {
```

Comment:
why 2 space? please re-check for all

Reply:
style.css file is a demo file, I will delete it and edit other files. Thanks again for your feedback.